Documentation/Installation.rst

```diff
@@ -9,8 +9,8 @@ Building the package
 --------------------
 
 
-* Ensure that 'dpkg' and 'svn' packages are installed on the machine
-* Check out the root of the branch/tag/branch of MCVirt
+* Ensure the build dependancies are installed: ``dpkg, python-docutils``

```

Comment:
Is it worth mentioning that if python-docutils is not installed, that the build will still work, but the man pages won't be included?